lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
```Ruby
# Returns only object records
records(object)
[{:name => 'name_1, :id => 'hfhdh1223'},{...n-1}]

# Returns object info
records(records_info)
[{:total_pages=>999, :current_page=>1, :total_records=>}]
```
This enables a separation between the info of all the knackhq records and the records themselves. We don't care that record #30 is in the `current page=1` of `total_records=100`  first batch of received records from knackhq in our interactor, we only care that it is mirrored.  And if we do, then we have the method `records_by_page` which returns the objects' records by batches of 500.